pgtrigger/runtime.py

```diff
@@ -276,10 +307,10 @@ def schema(*schemas: str, databases: Union[List[str], None] = None):
         yield
 
 
-schema.session = _schema_session
+schema.session = _schema_session  # type: ignore
```

Comment:
Fix here is making these class-based context managers, can do so in a follow-up PR. Just didn't want to refactor them here.